### PR TITLE
[Foundation] Add 32 more FN Keys

### PR DESCRIPTION
**Pull request by LeoHChen**
- Add keys to list
- Create new version instead of editing v1
- Fix v2 configuration

### CODE REVIEW

**Comment by harmony-ek** [comment]
Trying to repro the test failure on local

**Comment by harmony-ek** [comment]
From [Travis raw log](https://api.travis-ci.org/v3/job/567646140/log.txt):

```
<deploy ek-800> (deploy-py37) ip-172-31-8-106 18:44:47 harmony ((cd7a2155...)) $ 1123 grep -v -e '^go: finding ' -e '^Parsing meta tags from ' -e '^Fetching ' -e '^get "[^"]*": found meta tag ' log.txt  | tail
ar r lib/libbls256.a obj/bls_c256.o
g++ -shared -o lib/libbls256.so obj/bls_c256.o -L/home/travis/gopath/src/github.com/harmony-one/bls/../mcl/lib -lmcl
ar: creating lib/libbls256.a
$ cd ../harmony
$ go get -v ./...
go: github.com/inconshreveable/mousetrap@v1.0.0: unknown revision v1.0.0
go: error loading module requirements
The command "go get -v ./..." failed and exited with 1 during .

Your build has been stopped.
```

**Comment by harmony-ek** [comment]
```
<deploy ek-800> (deploy-py37) ip-172-31-8-106 18:44:51 harmony ((cd7a2155...)) $ 1124 go mod why github.com/inconshrevable/mousetrap@v1.0.0
go: finding github.com/prometheus/prometheus/util/flock latest
go: finding github.com/prometheus/prometheus/util latest
# github.com/inconshrevable/mousetrap@v1.0.0
(main module does not need package github.com/inconshrevable/mousetrap@v1.0.0)
```

**Comment by harmony-ek** [comment]
v1.0.0 is actually there… Jenkins, test this please.  Third time is—hopefully—the charm.

**Comment by harmony-ek** [comment]
Actually not Jenkins... Travis LOL, triggering rebuild.

**Comment by harmony-ek** [comment]
Also running `travis_checker.sh` locally on my dev desktop.

File: internal/configs/sharding/mainnet.go [hunk 1]
```diff
@@ -10,6 +10,9 @@ const (
 	mainnetEpochBlock1 = 344064 // 21 * 2^14
 	blocksPerShard     = 16384  // 2^14
 	mainnetV1Epoch     = 1
+	mainnetV2Epoch     = 5
+
+	mainnetVdfDifficulty = 50000 // This takes about 100s to finish the vdf
```
Comment (rlan35) [inline internal/configs/sharding/mainnet.go]: Why is this VDF difficulty added to r3?

Comment (LeoHChen) [reply internal/configs/sharding/mainnet.go]: should remove it.

**Pull request merged**